action.yml

```diff
@@ -33,6 +33,8 @@ inputs:
   token:
     description: Sonarqube access token
     required: true
+  github-token:
```

Comment:
I don't actually think you need to pass this in separately, you should be able to use the one that is passed automatically in the environment (that's what I did for the github-script action too)

Reply:
... so https://docs.github.com/en/actions/reference/authentication-in-a-workflow in other words.

Reply:
Uuuh... the way I read it, it's specifically for github-script, it's not available in other actions afaict. You just got my hopes up! Damn you @callebjorkell

Reply:
Are you sure about that? You also get it passed on just the regular "run" stuff (so without actions)